workflow/drilling.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Drilling Into Your Data
+---
+
+The Drill Down feature enhances your ability to investigate, analyze, and resolve issues without needing to switch contexts. Instead of sifting through large volumes of data indiscriminately, focus your efforts on drilling into relevant and focused subsets of data in order to save your team time and effort.
+
+This feature is available across all of Middleware and works by passing context through from the beginning to the end of a request. Clicking on any of Middleware's interactive tables will produce popouts that divulge the interconnectedness of your system infrastructure in a single window. Middleware's interactive tables will produce popouts that allow you to drill down to root cause regardless of which direction you start from.
+
+# Metrics
+
+Metrics are compact statistical representations of system state and resource utilization. They are quantifiable data points within your software system, like throughput, resource utilization, and error rates.
+
+Metrics can be considered the starting point for all of your drill-downs. Metrics provide a high-level view of the system in question, whereas traces and logs provide you with a more granular look into your system-level data.
```

Comment:
"Metrics can be considered the starting point" isn't necessarily true. Some users will start from traces and others from logs. I think if you remove that sentence the rest of this paragraph works really well as is.